chips/cc26x2/src/crt1.rs

```diff
@@ -14,121 +9,13 @@ extern "C" {
     static mut _ezero: u32;
     static mut _srelocate: u32;
     static mut _szero: u32;
-    fn reset_handler();
+    pub fn reset_handler();
 
     // _estack is not really a function, but it makes the types work
     // You should never actually invoke it!!
-    fn _estack();
-}
-
-use crate::event_priority;
-macro_rules! generic_isr {
-    ($label:tt, $priority:expr) => {
-        #[cfg(target_os = "none")]
-        #[naked]
-        unsafe extern "C" fn $label() {
-            stash_process_state();
-            set_event_flag_from_isr($priority);
-            disable_specific_nvic();
-            set_privileged_thread();
-        }
-    };
-}
-
-macro_rules! custom_isr {
-    ($label:tt, $priority:expr, $isr:ident) => {
-        #[cfg(target_os = "none")]
-        #[naked]
-        unsafe extern "C" fn $label() {
-            stash_process_state();
-            set_event_flag_from_isr($priority);
-            $isr();
-            set_privileged_thread();
-        }
-    };
+    pub fn _estack();
 }
 
-use crate::event_priority::EVENT_PRIORITY;
-generic_isr!(gpio_nvic, EVENT_PRIORITY::GPIO);
-generic_isr!(i2c0_nvic, EVENT_PRIORITY::I2C0);
-generic_isr!(aon_rtc_nvic, EVENT_PRIORITY::AON_RTC);
```

Comment:
Don't believe these were added back to the new interrupt table

Reply:
Fixed and seems to resolve the test issue